features/achordion.c

```diff
@@ -21,6 +21,7 @@
  */
 
 #include "achordion.h"
+#include "timer.h"
```

Comment:
Nit: This include shouldn't be needed. The achordion.h header includes quantum.h, which includes timer.h.

Reply:
Done.